styles/python.md

```diff
@@ -234,27 +204,30 @@ Because Python's indentation style is unlike many C-based languages, your editor
 Examples of line splitting from our code
 
 ```py
-BAD:  zero_duration_videos = video_models.Video.all().filter("duration =", 0).fetch(10000)
+# BAD
+
+zero_duration_videos = video_models.Video.all().filter("duration =", 0).fetch(10000)
 
-GOOD: zero_duration_videos = (video_models.Video.all()
+# GOOD
+zero_duration_videos = (video_models.Video.all()
                         .filter("duration =", 0)
                         .fetch(10000))
 ```
 
 ```py
-# BAD:
+# BAD
 return current_app.response_class("OAuth error. %s" % e.message, status=401, headers=build_authenticate_header(realm="http://www.khanacademy.org"))
 
-# GOOD:
+# GOOD
 return current_app.response_class(
     "OAuth error. %s" % e.message, status=401,
     headers=build_authenticate_header(realm="http://www.khanacademy.org"))
 ```
 
 ```py
-# Bad:
+# BAD
 kwargs = dict((str(key), value) for key, value in topic_json.iteritems() if key in ['id', 'title', 'standalone_title', 'description', 'tags', 'hide'])
-# Good:
+# GOOD
 kwargs = dict((str(key), value)
```

Comment:
Sorry for the late response on this. We can discuss further in another PR, but yeah, something like that (I'd maybe put the list items in 2 rows, so the code block doesn't get too long, though there are advantages to having one item per line too).